### PR TITLE
Update schema

### PR DESCRIPTION
**Pull request by rly**
- Store probabilities as float instead of int
- Provide column names for results table

### CODE REVIEW

File: ml/ml_table.yaml [hunk 1]
```diff
@@ -71,24 +71,31 @@ groups:
...
+    data_type_inc: ClassProbability
     doc: a column to store the class probability for each class across the samples
     quantity: '?'
-  - data_type_inc: ClassLabel
+  - name: class_label
+    data_type_inc: ClassLabel
     doc: a column to store which class a sample was classified as
     quantity: '?'
-  - data_type_inc: RegressionOutput
+  - name: reg_output
```
Comment (ajtritt) [inline ml/ml_table.yaml:90]:
```suggestion
  - name: estimate
```

This I'm less certain of

**Review by ajtritt — CHANGES_REQUESTED** [review]
Thanks for doing this. I think we should change the TrainValidationTestMask to TrainValidationTestSplit. Same for the CV column. "Split" is more in line with ML lingo

File: ml/ml_table.yaml (hunk 1 above)
Comment (rly) [reply ml/ml_table.yaml:90]: How about prediction?

Comment (ajtritt) [reply ml/ml_table.yaml:90]: Yeah, I thought about that. But sometimes prediction gets used for class labels in classification

**Comment by rly** [comment]
> Thanks for doing this. I think we should change the TrainValidationTestMask to TrainValidationTestSplit. Same for the CV column. "Split" is more in line with ML lingo

Done

**Review by ajtritt — COMMENTED** [review]
Sorry, forgot to click "Submit review" last week.

File: ml/ml_table.yaml [hunk 2]
```diff
@@ -71,24 +71,31 @@ groups:
...
+    data_type_inc: CrossValidationSplit
     doc: a column to indicate which cross-validation fold a sample was part of
     quantity: '?'
-  - data_type_inc: ClassProbability
+  - name: class_proba
+    data_type_inc: ClassProbability
     doc: a column to store the class probability for each class across the samples
     quantity: '?'
-  - data_type_inc: ClassLabel
+  - name: class_label
```
[the remark this answers is not in the document]
Comment (rly) [reply ml/ml_table.yaml:86]: I like `predicted_class`

Comment (rly) [inline ml/ml_table.yaml:86]:
```suggestion
  - name: predicted_class
```

**Review by ajtritt — APPROVED** [review]

**Pull request merged**